docs/data/charts/tooltip/tooltip.md

```diff
@@ -53,6 +53,41 @@ This kind of interaction is controlled by series properties `highlightScope` whi
 
 {{"demo": "ElementHighlights.js"}}
 
+## Interaction
+
+### Element click
+
+Charts provide an `onClick` callback with the following signature: `(event, {item, axis}) => void`.
+The same behavior can be obtained with composition by using `<ChartsOnClickHandler onClick={clickHandler} />`.
+
+#### Data collection
+
+The `item` can contains two properties:
+
+- `series` The series definition of the item clicked.
+- `dataIndex` The index of the item in the series data. This property can be `undefined` if user click on a line, or an area.
+
+The `axis` contains three properties:
+
+- `dataIndex` The index in the axis data.
+- `axisValue` The value in the axis.
+- `seriesValues` The mapping of series ids to their values along this axis position.
+
+The next demo allows to experiment the data received by `onClick` with various charts.
+
+Notice that `axis` is not defined for pie and scatter charts.
+You can also get `axis` and no `item` in bar and line charts by clicking outside of elements.
+
+{{"demo": "ElementClickNoSnap.js"}}
+
+#### Event management
+
+The `event` is the "onClick" event received by the `<svg/>` element.
```

Comment:
The following is closer to the actual behavior right?

```suggestion
The `event` is the "onClick" event emitted by the `<svg/>` element.
```

But I think we could rework this section a bit to be clearer.